mycroft/configuration/mycroft.conf

```diff
@@ -268,11 +268,11 @@
   // - https://github.com/OpenVoiceOS/OVOS-local-backend
   // - https://mycroft-ai.gitbook.io/docs/using-mycroft-ai/pairing-your-device
   "server": {
-    "disabled": true,
-    "url": "https://api.mycroft.ai",
+    "backend_type": "offline",
```

Comment:
Valid options should be documented here, similar to VAD

Reply:
will add options, as a side note this should also be updated whenever ovos-backend-client version is bumped and supports a new backend

Reply:
Maybe there should just be a comment here that refers to the backend client readme?